components/supervisor-api/status.proto

```diff
@@ -139,6 +140,11 @@ message ExposedPortInfo {
     // action hint on expose
     OnPortExposedAction on_exposed = 3;
 }
+message TunneledPortInfo {
+  uint32 target_port = 1;
```

Comment:
Please add comments that explain what these fields mean from which perspective, i.e. that the `target_port` is the desired port of the tunnel on the remote machine (seen from the workspace's perspective).